db/migrate/20220705211412_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :users, null: false, foreign_key: true
```

Comment:
- [ ] Let's reference author, this way we can achieve the `author_id` shown on the ERD diagram. If we look at the ERD we can't see `user_id` as foreign_key but `author_id`